packages/@react-types/combobox/src/index.d.ts

```diff
@@ -48,12 +48,14 @@ export interface ComboBoxProps<T> extends CollectionBase<T>, SingleSelection, In
   shouldFlip?: boolean
 }
 
-export interface SpectrumComboBoxProps<T> extends ComboBoxProps<T>, SpectrumLabelableProps, StyleProps {
+export interface SpectrumComboBoxProps<T> extends ComboBoxProps<T>, SpectrumLabelableProps, StyleProps, Omit<AsyncLoadable, 'isLoading'> {
   /** Whether the ComboBox should be displayed with a quiet style. */
   isQuiet?: boolean,
   /**
    * Direction the menu will render relative to the ComboBox.
    * @default 'bottom'
    */
-  direction?: 'bottom' | 'top'
+  direction?: 'bottom' | 'top',
+  /** The current loading state of the ComboBox. Determines whether or not the progress circle should be shown. */
+  loadingState?: 'loading' | 'sorting' | 'loadingMore' | 'error' | 'idle' | 'filtering'
```

Comment:
Feels odd to have all of these extra states when ComboBox only uses `loading`/`loadingMore`/`filtering`, but gotta  match the type that `state` has from `useAsyncList`. Actually might be nice to have validationState = invalid if `error` is true in `loadingState` perhaps

Reply:
I think it's ok. We may want to use more of these in the future (e.g. error states). Maybe we could move the type into a shared type?